telegram/document.py

```diff
@@ -41,13 +41,14 @@ class Document(TelegramObject):
 
     """
 
-    def __init__(self, file_id, thumb=None, file_name='', mime_type='', file_size=0, **kwargs):
+    def __init__(self, file_id, thumb=None, file_name=None, mime_type=None, file_size=0, **kwargs):
         # Required
         self.file_id = str(file_id)
         # Optionals
         self.thumb = thumb
         self.file_name = file_name
-        self.mime_type = str(mime_type)
+        if mime_type:
+            self.mime_type = str(mime_type)
```

Comment:
Please make sure `self.mime_type` is always available. I think that this should be sufficient (no need to use `str()` here):
`self.mime_type = mime_type` 
If you still think `str()` is required, you can do:
`self.mime_type = str(mime_type) if mime_type is not None else None`